packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -88,6 +91,22 @@ public void setTVPreferredFocus(ReactViewGroup view, boolean hasTVPreferredFocus
     }
   }
 
+  @ReactProp(name = ViewProps.BACKGROUND_IMAGE, customType = "BackgroundImage")
+  public void setBackgroundImage(ReactViewGroup view, @Nullable ReadableArray backgroundImage) {
```

Comment:
After this lands, this change will intersect a bit with another one, where I have been refactoring to be able to eventually apply background styles as part of BaseViewManager. https://github.com/facebook/react-native/pull/45832/commits/bbcca54e78cb15097baa6634940de5fc8237b2a3

This also enables the infra for `CompositeBackgroundDrawable`, for any future cases where we want to use individual Drawables as units of layering (like, if we wanted to mix Fresco provided bitmap image drawables, between gradients, or something like that.

Reply:
`CompositeBackgroundDrawable` would be cool!

Reply:
It is kind of similar to iOS view layers. Exciting!